data/json/mapgen/animalpound.json

```diff
@@ -0,0 +1,161 @@
+[
+  {
+    "name": "GROUP_ANIMALPOUND_CATS",
+    "type": "monstergroup",
+    "default": "mon_null",
+    "monsters": [
+      { "monster": "mon_cat", "freq": 300, "cost_multiplier": 0 }
+    ]
+  },
+  {
+    "name": "GROUP_ANIMALPOUND_DOGS",
+    "type": "monstergroup",
+    "default": "mon_dog_zombie_rot",
+    "monsters": [
+      { "monster": "mon_zombie_dog", "freq": 250, "cost_multiplier": 0 },
+      { "monster": "mon_dog_zombie_rot", "freq": 175, "cost_multiplier": 0 },
+      { "monster": "mon_dog", "freq": 75, "cost_multiplier": 0 },
+      { "monster": "mon_dog_bull", "freq": 75, "cost_multiplier": 0 },
+      { "monster": "mon_dog_pitbullmix", "freq": 100, "cost_multiplier": 0 },
+      { "monster": "mon_dog_chihuahua", "freq": 75, "cost_multiplier": 0 },
+      { "monster": "mon_dog_gshepherd", "freq": 35, "cost_multiplier": 0 }
+    ]
+  },
+  {
+    "id": "animalpound",
+    "type": "overmap_terrain",
```

Comment:
Not saying there aren't any, but I haven't seen any mapgen files with the overmap_terrain defined within them. I doubt it would cause an error but for organizational purposes I _may_ be better to define them in overmap_terrain.json.